packages/webapp/src/app/staticdiff.ts

```diff
@@ -6,9 +6,9 @@ import {
   saveAs
 } from 'file-saver';
 
-import {
+/* import {
   EditorWidget
-} from 'nbdime/lib/common/editor';
+} from 'nbdime/lib/common/editor'; */
```

Comment:
Commented code to be removed

Reply:
@krassowski Thanks. This will be removed.